Makefile.am

```diff
@@ -6,6 +6,7 @@
 # as published by the Free Software Foundation.
 #
 
+ACLOCAL_AMFLAGS = -I m4
```

Comment:
Could you just explain in the commit message what the reason for this change is?